flaml/automl.py

```diff
@@ -2284,6 +2332,12 @@ def custom_metric(
                     period: int | forecast horizon for ts_forecast tasks.
                     gpu_per_trial: float, default = 0 | A float of the number of gpus per trial,
                     only used by TransformersEstimator and XGBoostSklearnEstimator.
+                    group_ids: list of strings of column names identifying a time series, only
+                    used for panel time series forecasting
+                    TemporalFusionTransformer: [TimeSeriesDataSet pytorchforecasting](https://pytorch-forecasting.readthedocs.io/en/stable/api/pytorch_forecasting.data.timeseries.TimeSeriesDataSet.html)
```

Comment:
With this line, "TemporalFusionTransformer" will be listed as a key in fit_kwargs. Please revise.